### PR TITLE
Add more documentation on specifics of our docker usage

### PR DESCRIPTION
**Pull request by cpennington**
@e0d, @feanil: Review?

### CODE REVIEW

**Comment by edx-webhook** [comment]
# PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)`

# Sandbox
- [ ] Build a sandbox for your branch and add a link here

# Testing
- [ ] i18n
- [ ] RTL
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Analytics
- [ ] Performance
- [ ] Database migrations are backwards-compatible

# Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review #1
- [ ] Code review #2
- [ ] UI strings/error msgs review
- [ ] UX review
- [ ] Accessibility review
- [ ] Product review

# DevOps assistance
If it's needed, or even if you suspect it might be needed, do any/all these things as early as possible. Don't surprise devops right before merging! For example, touching envs/aws.py or or envs/common.py are dead giveaways that you'll need assistance when pushing your changes out. Things that require devops attention include, but are not limited to:
- configuration changes
- feature flags
- migrations
Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
- [ ] File a support ticket

FYI: Tag anyone who might be interested in this PR here.

# Post-review
- [ ] Squash commits

**Comment by feanil** [comment]
:+1:

File: docker/README.md [hunk 1]
```diff
@@ -2,6 +2,34 @@
...
+
+`Dockerfile`s for individual services should be placed in
+`docker/build/<service>`. There should be an accompanying `ansible_overrides.yml`
+which specifies any docker-specific configuration values.
+
+Once the `Dockerfile` has been created, it can be built and published
+using a set of make commands.
+
+```shell
+make docker.build.<service>  # Build the service container (but don't tag it)
```
Comment (e0d) [inline docker/README.md:23]: Please clarify the relation between current config checkout the the artifact being built.

Comment (cpennington) [inline docker/README.md:23]: Will do

**Pull request merged**